themes/rr/rr.theme.sh

```diff
@@ -0,0 +1,29 @@
+#! bash oh-my-bash.module
+
+# rr is a simple one-liner prompt inspired by robbyrussell from ohmyzsh themes.
+#
+# Looks:
+#
+# ➜  anish ~ cd .bash-it/themes/dulcie
+# ➜  anish custom-dulcie git:(master ✓) # with git
+#
+# Configuration. Change these by adding them in your .bash_profile
+
+function _omb_theme_RR_PROMPT() {
+    # configs
+    user_name="${_omb_prompt_white}\u${_omb_prompt_reset_color}"
+    base_directory="${_omb_prompt_bold_blue}\W${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_PREFIX="${_omb_prompt_bold_purple}git:(${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_bold_purple})${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_CLEAN="${_omb_prompt_bold_green} ✓${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_DIRTY="${_omb_prompt_bold_red} ✗${_omb_prompt_reset_color}"
+    arrow="${_omb_prompt_bold_purple}➜${_omb_prompt_reset_color}"
```

Comment:
We want to make the variables used only in the present function call local.

```suggestion
    local user_name="${_omb_prompt_white}\u${_omb_prompt_reset_color}"
    local base_directory="${_omb_prompt_bold_blue}\W${_omb_prompt_reset_color}"
    local SCM_THEME_PROMPT_PREFIX="${_omb_prompt_bold_purple}git:(${_omb_prompt_reset_color}"
    local SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_bold_purple})${_omb_prompt_reset_color}"
    local SCM_THEME_PROMPT_CLEAN="${_omb_prompt_bold_green} ✓${_omb_prompt_reset_color}"
    local SCM_THEME_PROMPT_DIRTY="${_omb_prompt_bold_red} ✗${_omb_prompt_reset_color}"
    local arrow="${_omb_prompt_bold_purple}➜${_omb_prompt_reset_color}"
```